devices/common_patches/dbgmcu.yaml

```diff
@@ -0,0 +1,17 @@
+_include:
+ - dbgmcu_rename.yaml
+
+DBGMCU:
+  _modify:
+    DBGMCU_IDCODE:
+      name: IDCODE
+    DBGMCU_CR:
+      name: CR
+    DBGMCU_APB1_FZ:
+      name: APB1_FZ
+    DBGMCU_APB2_FZ:
+      name: APB2_FZ
```

Comment:
These could now be replaced with a `_strip: DBGMCU_` (since #187)

Reply:
Oh, yes, much nicer !